doc/about.rst

```diff
@@ -0,0 +1,15 @@
+About PyViz Topic Examples
+==========================
+
+PyViz Topic Examples is home for domain-specific narrative examples using
+multiple PyViz projects. Each project is isolated and fully described:
+runnable locally and deployable to Anaconda Enterprise.
+
+This work is supported and maintained by `Anaconda
+<https://www.anaconda.com>`_.  PyViz Topics is part of the `PyViz
+<https://pyviz.org>`_ project to make Python data visualization tools
+simpler and more powerful.
+
+The primary PyViz developers are Philipp Rudiger, James A. Bednar,
```

Comment:
```suggestion
The primary PyViz maintainers are Philipp Rudiger, James A. Bednar,
```

(on a site like this I'd call the authors of the notebooks the "developers"; we're just maintaining it.  (And the authors list will of course change, and should be on each notebook, so we don't need to try to cover that here.)